packages/terra-functional-testing/lib/test-runner/test-runner.js

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs');
+const path = require('path');
+const Launcher = require('@wdio/cli').default;
+
+class TestRunner {
+  /**
+   * Resolves the configuration file path.
+   * If no path is provided the current working directory will be searched. Falls back to the default configuration file.
+   * @param {string} configPath - The path to the test runner configuration file.
+   * @returns {string} A resolved path to the test runner configuration file.
+   */
+  static configPath(configPath) {
+    if (configPath) {
+      return path.resolve(configPath);
+    }
+
+    const relativeConfigPath = path.resolve(process.cwd(), 'wdio.conf.js');
+
+    if (fs.existsSync(relativeConfigPath)) {
+      return relativeConfigPath;
+    }
+
+    return path.resolve(__dirname, './wdio.conf.js');
+  }
+
+  /**
+   * Starts the test runner.
+   * @param {Object} options - The test run options.
+   * @returns {Promise} A promise that resolves with the test run exit code.
+   */
+  static async run(options) {
+    try {
+      const { config } = options;
+
+      const configPath = TestRunner.configPath(config);
+      const testRunner = new Launcher(configPath);
```

Comment:
as we add more cli flags in the future, will we pass options as the 2nd parameter?
```suggestion
      const testRunner = new Launcher(configPath, options);
```

Reply:
Yes, as more options are added they will be passed to the launcher. The options will need to be extracted and organized so we only pass options relevant to the wdio launcher. We don't want to pass the entire options parameter to the launcher as it will likely include custom options from our cli interface.